Chap_API_Struct.tex

```diff
@@ -623,6 +676,23 @@ \subsubsection{Process structure support macros}
 \argin{r}{Rank to be assigned (\refstruct{pmix_rank_t})}
 \end{arglist}
 
+\littleheader{Transfer a procID structure}
+\declaremacro{PMIX_XFER_PROCID}
+
+Transfer contents of one \refstruct{pmix_proc_t} value to another \refstruct{pmix_proc_t}.
```

Comment:
Are the semantics of transfer in this case basically just copy (i.e., the source structure is left untouched)? Or is the source structure modified in some way?

Reply:
Strictly a copy, consistent with the other 'xfer' macros